server/backend/opendistro_security_client.ts

```diff
@@ -136,9 +136,66 @@ export class SecurityClient {
 
   public async getTenantInfo(request: KibanaRequest) {
     try {
-      return this.esClient.asScoped(request).callAsCurrentUser('opendistro_security.tenantinfo');
+      return await this.esClient.asScoped(request).callAsCurrentUser('opendistro_security.tenantinfo');
     } catch (error) {
       throw new Error(error.message);
     }
   }
+
+  public async getSamlHeader(request: KibanaRequest) {
+    try {
+      const response = await this.esClient.asScoped(request).callAsCurrentUser('opendistro_security.authinfo');
+      // response is expected to be an error
+      throw Error('Invalid Saml configuration.');
```

Comment:
I didn't get the idea why we need to throw and catch the error in the same function. Could you provide some more information?

Reply:
this throw is no needed, the logic is that, call ES API, which expect an error, the error messages includes the direct location, and request ID of the SAML IDP